### PR TITLE
Replace Foundation/Bourbon with Synfrastructure

### PR DESCRIPTION
**Pull request by paulstatezny**
### Acceptance Criteria
1. The app UI looks like it does on `master`.

### Tasks
- Replace Foundation/Bourbon with Synfrastructure

### Additional Notes
- This is a blocker for #125. So branch this off of that issue and make this a pull into that issue.

### CODE REVIEW

**Comment by canfie1d** [comment]
:soon:

**Comment by canfie1d** [comment]
:star:

**Comment by paulstatezny** [comment]
Looks good from a dev perspective. :star2:

File: application/ui/scss/dependencies/_fonts.scss [hunk 1]
```diff
@@ -65,28 +65,13 @@ $inconsolata : 'inconsolata';
...
-
 .ital-text {
     font-style:italic;
     padding-right: 3px;
 }
 
 .ital-text__bold {
-    @extend .ital-text;
+    font-style:italic;
+    padding-right: 3px;
```
Comment (kylecesmat) [inline application/ui/scss/dependencies/_fonts.scss]: colon spacing throughout file

**Comment by kylecesmat** [comment]
:interrobang:

**Comment by canfie1d** [comment]
:star2:

**Comment by kylecesmat** [comment]
looks great :pray:

**Comment by pedringt** [comment]
:hand:

**Comment by pedringt** [comment]
Receive "no gulp file found" error after gulp watch. Did npm/bower install etc.

**Comment by pedringt** [comment]
:-1: #129

**Comment by canfie1d** [comment]
:star2:

**Comment by kylecesmat** [comment]
:pray:

**Comment by pedringt** [comment]
:hand:

**Comment by pedringt** [comment]
:-1: Error after npm start

**Comment by pedringt** [comment]
```
WARNING in ./application/ui/components/params-list.jsx
jshint results in errors
  Missing 'new' prefix when invoking a constructor. @ line 29 char 9
            StoreWatchMixin('RequestStore', 'ConfigStore', 'StripeStore'),
Child extract-text-webpack-plugin:
    chunk    {0} /Users/paigeedrington/Desktop/Synapse/lively/node_modules/extract-text-webpack-plugin /Users/paigeedrington/Desktop/Synapse/lively/node_modules/css-loader/index.js!/Users/paigeedrington/Desktop/Synapse/lively/node_modules/autoprefixer-loader/index.js!/Users/paigeedrington/Desktop/Synapse/lively/node_modules/sass-loader/index.js?outputStyle=nested&includePaths[]=/bower_components&includePaths[]=/node_modules!/Users/paigeedrington/Desktop/Synapse/lively/application/ui/scss/app.scss 109077 [rendered]
        [0] ./~/css-loader!./~/autoprefixer-loader!./~/sass-loader?outputStyle=nested&includePaths[]=/bower_components&includePaths[]=/node_modules!./application/ui/scss/app.scss 108725 {0} [built]
        [1] ./~/css-loader/cssToString.js 352 {0} [built]
```

**Comment by paulstatezny** [comment]
:star:

**Comment by canfie1d** [comment]
:pray:

**Comment by paulstatezny** [comment]
:pray:

**Comment by pedringt** [comment]
:+1:

**Pull request merged**